examples/experimental/fs/fs.js

```diff
@@ -13,8 +13,24 @@ let file;
 })();
 
 export default async function () {
+	// About information about the file
 	const fileinfo = await file.stat();
 	if (fileinfo.name != "bonjour.txt") {
 		throw new Error("Unexpected file name");
 	}
+
+	// Define a buffer of the same size as the file
+	// to read the file content into.
+	const buffer = new Uint8Array(fileinfo.size);
+
+	// Read the file's content into the buffer
+	const bytesRead = await file.read(buffer);
+
+	// Check that we read the expected number of bytes
```

Comment:
Does this mean that we want to make `read` always filling the full buffer? 

This is not how go io.Reader is usually defined to work and is not how deno's [`read`](https://deno.land/api@v1.36.3?s=Deno.FsFile#method_read_0) is. 

I do expect with the current implementation that to be true, but even so, maybe we should not say that this is how it is expected to work. And the current inline documentation suggests that this was the intention, but then this script depends on it actually reading the full bugger everytime.

In that case it might be better to have it retry reading until it reads the whole `fileinfo.size`?

Reply:
> Does this mean that we want to make read always fill the full buffer?

No, `read` behaves how I understand you suggest it works. It reads as many bytes as possible into the buffer, but that amount could be smaller than the buffer indeed. Also, the resulting bytes count can be `null`, like in Deno, to indicate EOF.

I agree that the example might have been confusing in that regards, and I adapted it accordingly, to reflect reading a file in a buffered way indeed 👍🏻